gtsfm/bundle/bundle_adjustment.py

```diff
@@ -167,14 +170,25 @@ def get_metrics_from_sfm_data(sfm_data: GtsfmData, suffix: str) -> List[GtsfmMet
         # filter the largest errors
         filtered_result = optimized_data.filter_landmarks(self.output_reproj_error_thresh)
 
-        if wTi_list_gt is not None:
+        if cameras_gt is not None:
             # align the sparse multi-view estimate after BA to the ground truth pose graph.
-            filtered_result = filtered_result.align_via_Sim3_to_poses(wTi_list_gt)
+            filtered_result = filtered_result.align_via_Sim3_to_poses(
+                wTi_list_ref=[cam.pose() for cam in cameras_gt] if cameras_gt else []
+            )
             ba_pose_error_metrics = metrics_utils.compute_ba_pose_metrics(
-                gt_wTi_list=wTi_list_gt, ba_output=filtered_result
+                gt_wTi_list=[cam.pose() for cam in cameras_gt], ba_output=filtered_result
             )
             ba_metrics.extend(metrics_group=ba_pose_error_metrics)
 
+            output_tracks_classification = track_utils.classify_tracks3d_with_gt_cameras(
```

Comment:
these are called exit_codes in the 2d case and classifications here. I think its better to use one terminology. exit codes? Or are they subtly different in some way?

Reply:
changed it to exit codes everywhere